src/config/entities/configuration.ts

```diff
@@ -18,6 +18,31 @@ export default () => ({
   auth: {
     token: process.env.AUTH_TOKEN,
   },
+  balances: {
+    balancesTtlSeconds: parseInt(process.env.BALANCES_TTL_SECONDS ?? `${300}`),
+    providers: {
+      valk: {
+        baseUri:
+          process.env.VALK_BASE_URI ||
+          'https://merlin-api-v1.cf/api/merlin/public',
+        apiKey: process.env.VALK_API_KEY,
+        chains: {
+          1: { chainName: 'eth' },
+          10: { chainName: 'op' },
+          100: { chainName: 'xdai' },
+          1101: { chainName: 'pze' },
+          1313161554: { chainName: 'aurora' },
+          137: { chainName: 'matic' },
+          324: { chainName: 'era' },
+          42161: { chainName: 'arb' },
+          42220: { chainName: 'celo' },
+          43114: { chainName: 'avax' },
+          56: { chainName: 'bsc' },
+          8453: { chainName: 'base' },
```

Comment:
Nit: are these defined by Valk as `chainName`s? They look like `shortName`s to me.

Reply:
I don't have an opinion here, `chainName` was chosen in order to align it with the existing `prices.chains` configuration. Valk refers to these values as `id` in their documentation. The values can be seen as a somewhat counterpart of our ERC-3770 `shortName` that we have in the Config Service, but there are differences in the values, I don't think they are following that convention. 

So as I said, no opinion about the property name, we could rename it to `shortName` both here and into `prices.chains` in a separate PR, but that wouldn't be very strict because as I commented none of them is following ERC-3770's `shortName` concept.